src/lib/remark-plugins/rewrite-static-assets.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * This is based on mktg-content-workflow's docs transformers
+ * https://github.com/hashicorp/mktg-content-workflows/blob/main/shared/transforms/rewrite-static-assets.ts
+ *
+ * It rewrites image paths to source from the tutorials repository
+ * via the GitHub CDN, which is provided by the assets API in mtkg-content-workflows
+ */
```

Comment:
Copied this directly [from Learn](https://github.com/hashicorp/learn/blob/master/lib/remark-plugins/rewrite-static-assets.ts)